automatic/autohotkey.install/tools/VERIFICATION.txt

```diff
@@ -24,4 +24,4 @@ Using AU:
    Get-RemoteChecksum https://github.com/Lexikos/AutoHotkey_L/releases/download/v1.1.36.02/AutoHotkey_1.1.36.02_setup.exe
 
 File 'license.txt' is obtained from:
-   https://github.com/AutoHotkey/AutoHotkey/blob/26daaddf714135a6fcc730fe250bd1d41a9d3c53/license.txt
+   https://github.com/AutoHotkey/AutoHotkey/blob/df84a3e902b522db0756a7366bd9884c80fa17b6/license.txt
```

Comment:
This change is unrelated to your PR. Can you remove it?

Reply:
Done & moved to https://github.com/chocolatey-community/chocolatey-packages/pull/2240